### ISSUE TITLE
Is there a way to make it a dinamic library?

### ISSUE THREAD

**Issue by lucampelli** [juan-cardelino/stasm#7]
Needed for a Java-JNI-C++ project for University...
Any help is appreciated.
Thanks!

**Comment by juan-cardelino** [comment]
Yes, I've done that in some platforms (Mac, Android and Linux). Just look in the CMakeLists.txt the list of sources that need to be compiled and make your own project with them. You can also try to modify our current CMakelists.txt to do so. Both ways will take work however.

**Comment by lucampelli** [comment]
Thanks!! It worked!!!
